app/assets/javascripts/components/nav_recipe.js.jsx.coffee

```diff
@@ -0,0 +1,21 @@
+class @NavRecipe extends React.Component
+  @propTypes =
+    recipe: React.PropTypes.object
+    removeRecipe: React.PropTypes.func
+
+  handleClick: =>
+    @props.removeRecipe(@props.recipe)
+
+  render: =>
+    recipe = @props.recipe
+    recipeUrl = "/recipes/#{recipe.get('id')}"
+    recipePhoto = @props.recipe.get('photo')
+    photoUrl = if recipePhoto then recipePhoto.filename.url else "http://via.placeholder.com/140x100"
```

Comment:
In an earlier commit (3bf8c99152071567e0026cb5d19176c7ddcde9fc), I replaced the placeholder.com placeholders with `<span class="photo photo-placeholder"></span>` (Those placeholder images don't really serve a purpose here; and this way I could match the gray — #ddd — to other elements in the design)